coalib/results/SourceRange.py

```diff
@@ -126,6 +126,29 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Default representation::
+            filename: FROM start_line:start_column TO end_line:end_column
+        If whole line is affected::
+            filename: FROM start_line TO end_line
+        If whole file is affected::
+            filename
```

Comment:
I've thought about those colons again, and I get permanently confused when seeing them, as normally they already indicate (line)ranges, e.g. `3:4` means for me line 3 to 4. I think we should change the format str either to
```
FROM L3C3 TO L10C7
# or
L3C3:L10C7
# or with more spaces if it's too unreadable
L3 C3 : L10 C7
```